homeassistant/components/netatmo.py

```diff
@@ -18,7 +18,7 @@
 
 REQUIREMENTS = [
     'https://github.com/jabesq/netatmo-api-python/archive/'
-    'v0.9.2.zip#lnetatmo==0.9.2']
+    'v0.9.2.zip#lnetatmo==0.9.2.1']
```

Comment:
Should the .zip also have its version number updated?

Reply:
I've pushed 2 changes to this branch

Should I make a new PR?

Reply:
I do not see two commits (but I do see two modified files).

It was merged, so you cannot extend this PR now.

Reply:
So, should i make a new PR?

Reply:
If you have more work that you want to contribute, yes it should go in a new PR.

Reply:
Yes